### ISSUE TITLE
Finetuning text generation models using CLM task

### ISSUE THREAD

**Issue by rkitichotkul** [EricFillion/happy-transformer#245]
Usually, text generation models are finetuned using the causal language modeling (CLM) task. I see that the associated code `happytransformer/gen/trainer.py` has a comment saying it is based on [1], which is a code for training with CLM, as expected. However, why is `mlm_probability` which is a parameter for the masked language modeling task (MLM) appears to be a parameter for finetuning text generation model? I would appreciate if this is clarified. Thanks!

[1] https://github.com/huggingface/transformers/blob/master/examples/pytorch/language-modeling/run_clm.py

**Comment by EricFillion** [comment]
Very nice catch! I'll remove mlm_probability for text generation. Thanks for pointing this out.
